scala2.13/tools/pom.xml

```diff
@@ -62,7 +62,7 @@
                         <executions>
                             <execution>
                                 <id>if_modified_files</id>
-                                <phase>verify</phase>
+                                <phase>package</phase>
```

Comment:
Ah, ok.  `if_modified_files` is checked in the `package` phase instead of `verify`.